sway-lsp/tests/fixtures/auto_import/Forc.toml

```
@@ -6,4 +6,4 @@ name = "auto_import"
 implicit-std = false
 
 [dependencies]
-std = { git = "https://github.com/FuelLabs/sway", tag = "v0.48.1" }
+std = { path = "../../../../sway-lib-std" }
```

```suggestion
std = { git = "https://github.com/FuelLabs/sway", tag = "v0.66.5" }
```

We ended up pinning these examples so it wouldn't require updates when std-lib had changes. Would it be possible to just pin it to the latest master?

---

Done.